### ISSUE TITLE
All sequences to "barcode_Unknown" fastq--not demultiplexed

### ISSUE THREAD

**Issue by Rhizae42** [Molmed/fastq_demux#13]
Sample sheet columns in the following order: sample ID, i7, i5

I've tried i5 as both the forward and reverse complement.

Sequences are paired end from NextSeq550. fastq.gz files include separate R1 and R2 files.

Why are they all going to barcode_Unknown fastq files?

**Comment by alexell22** [comment]
Hey! have you managed to solve this issue? I have the same thing :(

**Comment by Rhizae42** [comment]
Yes. The sequencer did not retain barcode information on the sequences. I
had to get a program that could open any size text document to see it

*Sarah Dean, PhD*

*Assay Development Scientist, Molecular Oncology*


*TriCore Reference Laboratories1001 Woodward Pl NEAlbuquerque NM 87102*

*Adjunct Faculty, Math Science and Engineering*

*Central New Mexico Community College*

*Max Salazar Hall 563*

*525 Buena Vista SE *
*Albuquerque NM, 87106*

*“Imagination is more important than knowledge. Knowledge is limited.
Imagination encircles the world.” -Albert Einstein*
